fhir-server/src/main/java/com/ibm/fhir/server/util/RestAuditLogger.java

```diff
@@ -368,29 +373,55 @@ public static void logBundle(HttpServletRequest request, Bundle requestBundle, B
                 for (Entry bundleEntry : requestBundle.getEntry()) {
                     if (bundleEntry.getRequest() != null && bundleEntry.getRequest().getMethod() != null) {
                         requestMethod = bundleEntry.getRequest().getMethod();
+                        boolean operation =  bundleEntry.getRequest()
+                                                .getUrl().getValue().contains("$");
                         switch (HTTPVerb.Value.from(requestMethod.getValue())) {
                         case GET:
-                            readCount++;
+                            if (operation) {
+                                executeCount++;
+                            } else {
+                                readCount++;
+                            }
                             break;
                         case POST:
-                            createCount++;
+                            if (operation) {
+                                executeCount++;
+                            } else {
+                                createCount++;
+                            }
                             break;
                         case PUT:
-                            updateCount++;
+                            if (operation) {
+                                executeCount++;
+                            } else {
+                                updateCount++;
+                            }
+                            break;
+                        case DELETE:
+                            if (operation) {
+                                executeCount++;
+                            } else {
```

Comment:
I don't think operations can be invoked via DELETE

Reply:
Yes, they can - I added it for bulkdata-status.